how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -71,23 +78,29 @@ EOT
 
 ### Configure Kubelet to use containerd
 
-In order to allow kubelet use containerd (using CRI interface) configure the service to
-point to containerd socket.
+In order to allow kubelet use containerd (using CRI interface), configure the service to
```

Comment:
In order to allow kubelet, use containerd (using CRI interface) and configure the service to